src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs

```diff
@@ -393,6 +402,39 @@ public void CopyTo(T[] array, int arrayIndex)
             Array.Copy(_items, 0, array, arrayIndex, _size);
         }
 
+        public void CopyTo(int sourceIndex, int count, Span<T> destination)
+        {
+            if (sourceIndex < 0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(sourceIndex), SR.ArgumentOutOfRange_NeedNonNegNum);
+            }
+
+            if (count < 0)
+            {
+                ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.count, ExceptionResource.ArgumentOutOfRange_NeedNonNegNum);
+            }
+
+            if (_size - sourceIndex < count)
+            {
+                ThrowHelper.ThrowArgumentException(ExceptionResource.Argument_InvalidOffLen);
+            }
+
+            if (count > destination.Length)
+            {
+                throw new ArgumentException(SR.Argument_DestinationTooShort, nameof(destination));
+            }
+
+            if (_size > 0 && count > 0)
+            {
+                Buffer.Memmove(ref destination[0], ref _items[sourceIndex], (nuint)count);
```

Comment:
Should I change it to:

Buffer.Memmove(ref destination._pointer.Value, ref Unsafe.Add(ref Unsafe.As<byte, T>(ref _items.GetRawSzArrayData()), sourceIndex), (nuint)count);

?